lib/components/HotkeyedDropdown.jsx

```diff
@@ -0,0 +1,64 @@
+
+import { Dropdown } from 'react-bootstrap';
+import Mousetrap from 'mousetrap';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+
+// Like react-bootstrap's `Dropdown`, but can receive an extra `hotkey` prop:
+// a key combination handled by `mousetrap` that will toggle the open state
+// of the dropdown.
+// Also handles focusing the drop-down on the hotkey.
+
+export default class HotkeyedDropdown extends Dropdown {
```

Comment:
I think `extends React.Component` should be ok here? Same for the `HotkeyedMenuItem`.